### DISCUSSION TITLE
多次测试结果不同应该以哪次为准呢

### DISCUSSION THREAD

**Discussion by paladin4fan** [HMBSbige/NatTypeTester#164] [Q&A] [answered]
内置的几个stun服务器都是这个情况
每次打开程序 每个节点第一次测试 会等待较长时间 local end自动改成192.168.xx.xx  然后结果是PortRestrictedCone
结果出来后local end就会变成0.0.0.0 之后再次测试同个节点就local end地址就不会变 保持0.0.0.0 然后马上得出FullCone 继续点击测试按钮依然会马上返回fullcone
重启软件后又会回到第一条的情况
所以想请教一下这个情况以哪一种结果为准呢

**Comment by HMBSbige** [comment, marked as answer]
如果你只是随便测测玩的，你就当第一次为准的吧

如果你真的很在意你的网络类型，建议了解并使用 RFC5389
